demo/pages/docs/jsx/enUS/index.md

```diff
@@ -23,7 +23,7 @@ export default defineComponent({
 
 ## Props look like @update:\*
 
-In naive-ui, all props look like `on-update:*` has a corresponding `onUpdate*` prop (since in JSX `onUpdate:*` is not a valid prop name).
+In naive-ui, all props look like `on-update:*` has a corresponding `onUpdate*` prop (since in JSX `on-update:*` is not a valid prop name).
```

Comment:
```suggestion
In naive-ui, all props look like `on-update:*` has a corresponding `onUpdate*` prop (since in JSX `on-update:*` and `onUpdate:*` is not a valid prop name).
```